.github/stale.yml

```diff
@@ -1,30 +1,23 @@
 # Number of days of inactivity before an issue becomes stale
-daysUntilStale: 15
+daysUntilStale: 30
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 5
 # Issues with these labels will never be considered stale
 exemptLabels:
   - "not stale"
-  - "good first issue 👍"
-  - "good for beginners 😃"
-  - "help wanted 🆘"
   - "Priority: High 🚨"
-  - "Showcase 💖"
-  - "Topic: Reviews ⭐️💖"
-  - "Status: Assigned ➡"
-  - "Status: Work in Progress 🛠"
-  - "Type: Bug 🐛"
-  - "Type: Documentation 📚"
-  - "Type: Feature 🚀"
 # Label to use when marking an issue as stale
 staleLabel: stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   Hi!
 
-  This issue hasn't seen any activity recently. We close inactive issues after 20 days to manage the volume of issues we receive.
+  This issue hasn't seen any activity recently. We close inactive issues after
+  20 days to manage the volume of issues we receive.
```

Comment:
This should be 35 due stale after 30 + 5 until close.

Reply:
Whoops, forgot to update the message! Nice catch